openapi/nytimes.mostpopular/types.bal

```diff
@@ -17,7 +17,7 @@
 public type ArticlewithcounttypeMedia record {
     string caption?;
     string copyright?;
-    ArticlewithcounttypeMediametadata[] mediaMetadata?;
+    ArticlewithcounttypeMediametadata mediaMetadata?;
```

Comment:
Unexpected change in the data type. `ArticlewithcounttypeMediametadata[]` -> `ArticlewithcounttypeMediametadata`.

Reply:
@kapilraajP any particular reason to manually update `mediaMetadata` to `ArticlewithcounttypeMediametadata[]` ? According to the OpenAPI file `mediaMetadata` is not an array.

Reply:
I also couldn't see any reason here to change that according to openapi.yaml

Reply:
While testing response consist of array of mediaMetada. That is why manually changed in this as far as i know